www/src/css/components/_view.scss

```diff
@@ -0,0 +1,71 @@
+@use '../var' as *;
+
+/**
+ * Main document view
+ */
+
+$top-padding: 24px;
+
+// Document view
+.snow-view__docs {
```

Comment:
@melissamcewen I made a `view` component that tries to manage the layout of the page. This handles the docs columns, as well as the mobile nav, because currently those are the same element.

We could probably go farther with this, but hopefully this simplifies things a bit—the goal is when the viewport shrinks, have everything we need in this one file for the site